aws-env.dev

```diff
@@ -0,0 +1,8 @@
+## This file store all environment variables for the DEV environment
+PROJECT = lambda2sqs
+AWS_REGION = ap-southeast-1
+DEPLOY_S3_PREFIX = lambda2sqs
+AWS_PROFILE = uneet-dev
+DEFAULT_SECURITY_GROUP = sg-d8dde8a6
+##PRIVATE_SUBNETS = Not Created yet.
```

Comment:
without the subnets, this patch will break a potentially working setup, isn't it best just to add them?